### PR TITLE
More styleguide components

### LINKED ISSUE #212: Update styleguide with new components
**[Story](https://18f.storiesonboard.com/m/fec/!card/dSDD9jsDpx8iGTnTwnAhdQ):** So that the team can design with confidence and consistency, audit and update the styleguide to make sure it encompasses and accurately reflects our patterns.

**Tasks:**
- [x] ~~Add markup for all partials~~ => See below
- [x] Add colors
- [x] ~~Audit for consistency~~ => Make new issues for visual and code consistency updates

**Add or update the following components:**
- [ ] ~~Calendar components~~ (move to modules)
- [ ] ~~Charts~~ (deferring to later)
- [x] Contact items
- [x] Cycle select
- [x] Downloads
- [x] Dropdowns
- [x] Icon headings
- [ ] ~~Maps~~ (deferring to later)
- [x] Options
- [x] Overlay
- [x] Search results
- [x] Side nav
- [x] Table styles

### PR DESCRIPTION
**Pull request by noahmanger**
This begins the work of updating the style code documentation. Along the way, I'm doing some small clean up, as well, which will require small updates to markup in certain places. 

Resolves #212 

## Update:
This is ready for review. It contains a few kinds of changes:
- Adding or updating styleguide markup for existing components
- Moving some "components" into "modules"
- Updating the visual and interaction design of the styleguide itself slightly

There's more work to do, and I captured those issues separately, but considering this is so big already, would be good to get this reviewed and merged.

## Goes with:
https://github.com/18F/fec-cms/pull/267
https://github.com/18F/openFEC-web-app/pull/1164

### CODE REVIEW

**Comment by noahmanger** [comment]
The conflicts should be resolved. I'm not seeing any conflicts locally, so I think github might just be confused.

**Comment by noahmanger** [comment]
As far as review, it might be good for @jenniferthibault to take a look at things for a) how things show up on the styleguide and b) general understanding of what's going on.

And then either @ccostino or @adborden could review.

Happy to walk through what's going on here with one or all of you, too. It seems like a lot, but it's mostly just moving things around and adding comments.

**Comment by jenniferthibault** [comment]
Diving into check things out now.

**Comment by jenniferthibault** [comment]
K, I added a few new instances of adjusting the letter-spacing of Karla. 

screen shot 2016-04-07 at 1 52 06 pm
screen shot 2016-04-07 at 1 54 56 pm
screen shot 2016-04-07 at 1 58 34 pm
screen shot 2016-04-07 at 1 58 42 pm
screen shot 2016-04-07 at 2 04 14 pm
screen shot 2016-04-07 at 2 20 55 pm

The last one I'd like to include in this PR is on the calendar event details on the grid and list view, but wasn't sure what the best way to do that was. @noahmanger could you make these adjustments?

screen shot 2016-04-07 at 2 56 00 pm
screen shot 2016-04-07 at 2 56 15 pm
screen shot 2016-04-07 at 2 56 36 pm

One other thing on the style guide itself is that the subnav links are still broken. http://recordit.co/WuoWBcc0J7
Did you want to fix those here?

**Comment by noahmanger** [comment]
Got it!

File: fec-template/public/init.js [hunk 1]
```diff
@@ -26,3 +27,39 @@ new feedback.Feedback();
...
 new typeaheadFilter.TypeaheadFilter('.js-typeahead-filter', typeahead.datasets.committees);
+
+$('.js-accordion').each(function(){
+  var contentPrefix = $(this).data('content-prefix') || 'accordion';
+  var openFirst = $(this).data('open-first') || false;
+  var opts = {
+    contentPrefix: contentPrefix,
+    openFirst: openFirst
+  };
+  new accordion.Accordion({}, opts);
```
Comment (adborden) [inline fec-template/public/init.js]: houndci has a point, it seems a bit weird to use `new` here. But it looks like [aria-accordian](https://github.com/stephenmathieson/aria-accordian/blob/master/index.js#L25) does handle the `new` for you.

Maybe we should disable this rule?

Comment (noahmanger) [inline fec-template/public/init.js]: Yeah I think we should disable it. It's how we've been initializing our components elsewhere.

**Pull request merged**